### DISCUSSION TITLE
"--framework" flag confusion

### DISCUSSION THREAD

**Discussion by ES0T** [waf/CSharpRepl#269] [General]
Could you please provide more clarification or detail about the "--framework" flag ? Either just in this discussion thread or in the Readme/documentation itself.

I see there are 3 options :

1. Microsoft.NETCore.App (default)
2. Microsoft.AspNetCore.App
3. Microsoft.WindowsDesktop.App

What exactly is the difference between setting these different values? Is using "AspNetCore.App" basically just the same as using the default and then adding a bunch of "#r" using references that are required for ASP NET Core?

Thanks!

**Comment by waf** [comment]
It's essentially a set of assemblies. There's a great deepdive into it available here: https://natemcmaster.com/blog/2018/08/29/netcore-primitives-2/
